src/none.js

```diff
@@ -0,0 +1,28 @@
+var _curry2 = require('./internal/_curry2');
+
+
+/**
+ * Returns `true` if no elements of the list match the predicate,
+ * `false` otherwise.
+ *
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig (a -> Boolean) -> [a] -> Boolean
+ * @param {Function} fn The predicate function.
+ * @param {Array} list The array to consider.
+ * @return {Boolean} `true` if the predicate is not satisfied by every element, `false` otherwise.
+ * @example
+ *
+ *      R.none(isNaN)([1, 2, 3, NaN]); //=> false
+ *      R.none(isNaN)([1, 2, 3]); //=> true
```

Comment:
I like your simplification of the examples. :)

I think it would read better with the `true` example before the `false` one, but I don't know *why* I think this. Also, it would be clearer to provide both arguments at once.

     *      R.none(isNaN, [1, 2, 3]); //=> true
     *      R.none(isNaN, [1, 2, 3, NaN]); //=> false

Aside from this niggle, the pull request looks great!